src/config.cpp

```
@@ -18,11 +18,14 @@ namespace hdt
 					SkyrimPhysicsWorld::get()->getSolverInfo().m_numIterations = btClamped(reader.readInt(), 4, 128);
-				} else if (reader.GetLocalName() == "groupIterations") {
-					ConstraintGroup::MaxIterations = btClamped(reader.readInt(), 0, 4096);
-				} else if (reader.GetLocalName() == "groupEnableMLCP") {
-					ConstraintGroup::EnableMLCP = reader.readBool();
-				} else if (reader.GetLocalName() == "erp") {
+				} 
+				// This has been dead code for years. Todo: Remove references to this in all the configs/menus.
+				//else if (reader.GetLocalName() == "groupIterations") {
+				//	ConstraintGroup::MaxIterations = btClamped(reader.readInt(), 0, 4096);
+				//} else if (reader.GetLocalName() == "groupEnableMLCP") {
+				//	ConstraintGroup::EnableMLCP = reader.readBool();
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)